application/common/mail/new-user.html.php

```diff
@@ -13,10 +13,10 @@
 <p>
   The <?= Html::encode($organizationName) ?> IDP account you requested for
     <?php if (empty($user->getDisplayName())) {
-    echo Html::encode($user->getFirstName() . ' ' . $user->getLastName());
```

Comment:
Would be interesting to use `<?=` instead of `<?php` to avoid the echo, but not strictly necessary.

Reply:
That's kind of out of scope for this PR.  All those whitespace changes were done automatically.